README.md

```diff
@@ -70,22 +70,19 @@ Testground aims to be tightly integrated with the software engineering practices
 
 The current Testground Team is composed of:
 
-- @raulk - Lead Architect, Engineer, Developer
-- @daviddias - Engineer, Developer, acting as interim PM for the project
-- @nonsense - Engineer, Developer, Testground as a Service / Infrastructure Lead
-- @jimpick - Engineer, Developer
-- @stebalien - Engineer, Developer
-- @hacdias - Engineer, Developer
+- @raulk - Architect, Lead Software Engineer
+- @nonsense - Software Engineer, Testground as a Service / Infrastructure Lead
+- @fabiomartins91 - [Technical Project Manager (TPM)](https://github.com/ipfs/team-mgmt/blob/master/TEAMS_ROLES_STRUCTURES.md#working-group-technical-project-manager-tpm)
+- @hacdias - Software Engineer
+- @daviddias - ?? Figuring it out :D
```

Comment:
I'd rather we don't merge that ;)

You are on the team in some capacity, if it is not clear what, its okay, just add yourself as an Engineer or PM.

This list is not exhaustive anyways, because there are many other contributors, so it doesn't have to reflect the team structure all the time super precisely in my opinion.